common-rx/build.gradle

```diff
@@ -0,0 +1,17 @@
+apply plugin: 'java-library'
+apply plugin: 'kotlin'
+
+dependencies {
+    // noinspection DifferentStdlibGradleVersion
+    implementation(
+        "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
+    )
+
+    api(
+        "io.reactivex.rxjava2:rxjava:$rxjava_version",
+        "io.reactivex.rxjava2:rxkotlin:$rxkotlin_version"
+    )
+}
+
+targetCompatibility = JavaVersion.VERSION_1_8
+sourceCompatibility = JavaVersion.VERSION_1_8
```

Comment:
Is this doing anything? 🙈 

Android studio tells me "Assignment is not used"

Reply:
@Tuubz my AS doesn't show anything (including build) 😳

Reply:
lol, I synced the project again. Seems to be fine 🙇